tests/manager/manager-executeWriteCommand_error-002.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+MongoDB\Driver\Manager::executeWriteCommand() throws CommandException for invalid writeConcern
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php NEEDS('REPLICASET'); CLEANUP(REPLICASET); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(REPLICASET);
+
+$document = ['foo' => ['bar']];
+
+$command = new MongoDB\Driver\Command([
+    'findAndModify' => COLLECTION_NAME,
+    'query' => ['_id' => 'foo'],
+    'update' => $document,
```

Comment:
This can just be a literal `['foo' => ['bar']]` if we can remove the `$document` variable above.